docs/reference/migration/migrate_8_0/_cluster-node-setting-changes.asciidoc

```diff
@@ -0,0 +1,891 @@
+- `discovery.zen.fd.connect_on_network_disconnect`
+- `discovery.zen.fd.ping_interval`
+- `discovery.zen.fd.ping_timeout`
+- `discovery.zen.fd.ping_retries`
+- `discovery.zen.fd.register_connection_listener`
+- `discovery.zen.join_retry_attempts`
+- `discovery.zen.join_retry_delay`
+- `discovery.zen.join_timeout`
+- `discovery.zen.max_pings_from_another_master`
+- `discovery.zen.send_leave_request`
+- `discovery.zen.master_election.wait_for_joins_timeout`
+- `discovery.zen.master_election.ignore_non_master_pings`
+- `discovery.zen.publish.max_pending_cluster_states`
+
+*Impact* +
+Discontinue use of the `discovery.zen` settings. Specifying these settings in
+`elasticsearch.yml` will result in an error on startup.
+====
+
+.`http.content_type.required` has been removed.
+[%collapsible]
+====
+*Details* +
+The `http.content_type.required` setting was deprecated in Elasticsearch 6.0
+and has been removed in Elasticsearch 8.0. The setting was introduced in
+Elasticsearch 5.3 to prepare users for Elasticsearch 6.0, where content type
+auto detection was removed for HTTP requests.
+
+*Impact* +
+Discontinue use of the `http.content_type.required` setting. Specifying this
+setting in `elasticsearch.yml` will result in an error on startup.
+====
+
+.`http.tcp_no_delay` has been removed.
+[%collapsible]
+====
+*Details* +
+The `http.tcp_no_delay` setting was deprecated in 7.x and has been removed in 8.0. It has been replaced by
+`http.tcp.no_delay`.
+
+*Impact* +
+Use the `http.tcp.no_delay` setting. Discontinue use of the `http.tcp_no_delay`
+setting. Specifying the `http.tcp_no_delay` setting in `elasticsearch.yml` will
+result in an error on startup.
+====
+
+.`network.tcp.connect_timeout` has been removed.
+[%collapsible]
+====
+*Details* +
+The `network.tcp.connect_timeout` setting was deprecated in 7.x and has been removed in 8.0. This setting
+was a fallback setting for `transport.connect_timeout`.
+
+*Impact* +
+Use the `transport.connect_timeout` setting to change the default connection
+timeout for client connections. Discontinue use of the
+`network.tcp.connect_timeout` setting. Specifying the
+`network.tcp.connect_timeout` setting in `elasticsearch.yml` will result in an
+error on startup.
+====
+
+.`node.max_local_storage_nodes` has been removed.
+[%collapsible]
+====
+*Details* +
+The `node.max_local_storage_nodes` setting was deprecated in 7.x and
+has been removed in 8.0. Nodes should be run on separate data paths
+to ensure that each node is consistently assigned to the same data path.
+
+*Impact* +
+Discontinue use of the `node.max_local_storage_nodes` setting. Specifying this
+setting in `elasticsearch.yml` will result in an error on startup.
+====
+
+[[accept-default-password-removed]]
+.The `accept_default_password` setting has been removed.
+[%collapsible]
+====
+*Details* +
+The `xpack.security.authc.accept_default_password` setting has not had any affect
+since the 6.0 release of {es}. It has been removed and cannot be used.
```

Comment:
```suggestion
The `xpack.security.authc.accept_default_password` setting has not had any affect
since the 6.0 release of {es} and is no longer allowed.
```